wasm2c/wasm-rt-mem-impl.c

```diff
@@ -25,7 +25,7 @@
 #include <sys/mman.h>
 #endif
 
-#define PAGE_SIZE 65536
+#define WASM_RT_PAGE_SIZE 65536
```

Comment:
How about WASM_PAGE_SIZE.. since otherwise it might be confused with the host (runtime) page size?

Reply:
Done.